ipld/src/selector/mod.rs

```diff
@@ -168,13 +168,13 @@ impl SubAssign<u64> for RecursionLimit {
 /// Condition is expresses a predicate with a boolean result.
 ///
 /// Condition clauses are used several places:
-///   - in Matcher, to determine if a node is selected.
-///   - in ExploreRecursive, to halt exploration.
-///   - in ExploreConditional,
+///   - in `Matcher`, to determine if a node is selected.
+///   - in `ExploreRecursive`, to halt exploration.
+///   - in `ExploreConditional`,
 ///
 ///
-/// TODO -- Condition is very skeletal and incomplete.
-/// The place where Condition appears in other structs is correct;
+/// Condition is very skeletal and incomplete.
```

Comment:
The TODO comment is valid and shouldn't be removed. It should just be changed from a documentation comment to a normal comment.

Comments that start with three slashes are used for documentation. Comments that start with 2 slashes are used for internal development. TODO comments are internal and should never be part of the documentation.

Reply:
Re-added